examples/enqueue_url_scan.py

```diff
@@ -0,0 +1,69 @@
+# Copyright © 2019 The vt-py authors. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import asyncio
+import os
+import sys
+import vt
+
+
+async def get_urls_to_enqueue(queue, path, url):
+  """Finds which URLs will be enqueued to scan in VirusTotal."""
+  if url:
+    await queue.put(url)
+    return
```

Comment:
argparse sets `path` to None if it's not provided and that value is used as parameter in the function. Also, `path` is not a list but a file object so changing its type to a list (even though it would be None in this case) is more confusing for me rather than just returning.